cmd/snap-seccomp/main_test.go

```diff
@@ -233,7 +444,7 @@ func (s *snapSeccompSuite) TestCompile(c *C) {
 		{"read\nwrite\nexecve\n", "write", main.SeccompRetAllow},
 
 		// trivial denial
-		{"read", "execve", main.SeccompRetKill},
+		{"read", "ioctl", main.SeccompRetKill},
```

Comment:
Execve is now part of the `common` syscalls, i.e. it must be allowed so that we can actually run these tests here.